### PR TITLE
python.d/nvidia_smi: add bar1 chart

### PR DESCRIPTION
**Pull request by pbouchez**
##### Summary

This PR adds BAR1 memory usage chart. Bar1 memory already exists in nvidia_smi, it's just not displayed with Netdata.

##### Component Name

collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

##### Test Plan

Successfull build and running in production in our organization for more than 3 weeks (Ubuntu 18.04, Netdata 1.32.1).

##### Additional Information

image
image

### CODE REVIEW

File: collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py [hunk 1]
```diff
@@ -95,6 +97,13 @@ def gpu_charts(gpu):
                 ['fb_memory_used', 'used'],
             ]
         },
+        BAR_USAGE: {
+            'options': [None, 'Bar1 Memory Usage', 'MiB', fam, 'nvidia_smi.bar1_allocated', 'stacked'],
```
Comment (ilyam8) [inline collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py:101]:
```suggestion
            'options': [None, 'Bar1 Memory Usage', 'MiB', fam, 'nvidia_smi.bar1_memory_usage', 'stacked'],
```

Let's make the context more clear

**Comment by ilyam8** [comment]
Thanks for the contribution, @pbouchez. Looks good to me, I have only one minor suggestion.

File: collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
 MEM_UTIL = 'mem_utilization'
 ENCODER_UTIL = 'encoder_utilization'
 MEM_USAGE = 'mem_usage'
+BAR_USAGE = 'bar1_usage'
```
Comment (ilyam8) [inline collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py:31]:
```suggestion
BAR_USAGE = 'bar1_mem_usage'
```

File: collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py (hunk 1 above)
Comment (btravouillon) [reply collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py:101]: Or `bar1_memory_allocated` ? The above **MEM_USAGE** makes use of `nvidia_smi.memory_allocated`.

Comment (pbouchez) [reply collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py]: I just commited with `bar1_memory_usage`, but as you want!

Comment (ilyam8) [reply collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py]: > The above MEM_USAGE makes use of nvidia_smi.memory_allocated

Oh, you are right. I think that "mem_usage" is a better name, but we could make it "bar1_memory_allocated" to keep consistent with MEM_USAGE. I guess I'm ok with both variants.

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**